USA/CA/golden_west_college/url_name.txt

```diff
@@ -0,0 +1,5 @@
+https://www.goldenwestcollege.edu/../../Links/pdf/RELEASE_20190924_T141046_Coast_Community_College_Police_Department_Protocol_Manual.pdf, RELEASE_20190924_T141046_Coast_Community_College_Police_Department_Protocol_Manual.pdf
+https://www.goldenwestcollege.edu/../../Links/pdf/Golden_West_College_Crime_Statistics_2017-2019-.pdf, Golden_West_College_Crime_Statistics_2017-2019-.pdf
+https://www.goldenwestcollege.edu/../../Links/pdf/2020-Annual-Security-and-Fire-Safety-Report-ASFSR---Coast-Community-Colleges-Combined.pdf, 2020-Annual-Security-and-Fire-Safety-Report-ASFSR---Coast-Community-Colleges-Combined.pdf
+https://www.goldenwestcollege.edu/../../Links/pdf/2019-Annual-Security-Report-ASR.pdf, 2019-Annual-Security-Report-ASR.pdf
+https://www.goldenwestcollege.edu/../../Links/pdf/2018-Annual-Security-Report.pdf, 2018-Annual-Security-Report.pdf
```

Comment:
Did you mean to commit this? I thought these were generated

Reply:
Nope, forgot to delete.